src/test/regress/sql/alter_database_propagation.sql

```diff
@@ -56,4 +48,49 @@ alter database regression set lock_timeout from current;
 alter database regression set lock_timeout to DEFAULT;
 alter database regression RESET lock_timeout;
 
+set citus.enable_create_database_propagation=on;
+create database regression2;
+alter database regression2 with CONNECTION LIMIT 100;
+alter database regression2 with IS_TEMPLATE true CONNECTION LIMIT 50;
+alter database regression2 with IS_TEMPLATE false;
+
+
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :worker_1_port
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts4'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :worker_2_port
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts5'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :master_port
+
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = '%ALTER DATABASE%';
+
+alter database regression2 set TABLESPACE alter_db_tablespace;
+
+set citus.enable_create_database_propagation=on;
+alter database regression2 rename to regression3;
```

Comment:
Can we also go the worker node, and check datallowconn, datconnlimit, datistemplate, dattablespace values are correct for database regression3? I know that we are logging the remote commands, but this is extra check on it.

Reply:
I think we need to define our standards. In one of my previous PR's I checked all the nodes and @JelteF made me remove all the checks and told me just adding logs was sufficient. Adding both logs and checks is unnecessary IMO.

Reply:
Hmmm I understand, in that case let's go with Jelte's suggestion for now - i.e. keeping only the logs.

I am really pro the logs. And, as long as we don't get any errors, it means the command has been executed. But the reason I wanted the extra check is to make sure that Citus isn't intercepting the command in an unexpected way that results into an unexpected execution result. I realize this is extra-cautious, but maybe something to think about @JelteF